moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -88,6 +88,7 @@ class MoveItErrorCode : public moveit_msgs::MoveItErrorCodes
   }
 };
 
+MOVEIT_CLASS_FORWARD(MoveItErrorCode);
```

Comment:
I don't think this goes here. Can you motivate this?

Reply:
I don't think it is.